nightingale_manipulation/src/nightingale_manipulation/manipulation_action_client.py

```diff
@@ -127,37 +127,37 @@ def __init__(self):
             [closed, closed, closed], CFG["left_gripper"]["joints"]
         )
 
-        self.gripper_lock = {"left": 0, "right": 0}
+        self.gripper_lock = {"left": True, "right": True}
         self.unlock_left_gripper()
         self.unlock_right_gripper()
 
     def cmd_right_gripper(self, goal, blocking=True):
-        if self.gripper_lock[0]:
+        if self.gripper_lock["right"]:
             self.right_gripper.send_goal(goal)
             if blocking:
                 return self.right_gripper.wait_for_result()
             return True  # TODO: action_server.get_result()
         return False
 
     def cmd_left_gripper(self, goal, blocking=True):
-        if self.gripper_lock[1]:
+        if self.gripper_lock["left"]:
             self.left_gripper.send_goal(goal)
             if blocking:
                 return self.left_gripper.wait_for_result()
             return True  # TODO: action_server.get_result()
         return False
 
     def lock_right_gripper(self):
-        self.gripper_lock["right"] = 0
+        self.gripper_lock["right"] = True
 
     def unlock_right_gripper(self):
-        self.gripper_lock["right"] = 1
+        self.gripper_lock["right"] = False
```

Comment:
Shouldn't 1 be true and 0 be false?

Reply:
i think it makes sense for true to mean that the gripper is locked. not sure why gripper locked used to be 0 but i think its good now